src/commands/files/zip.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const BoxCommand = require('../../box-command');
+const { flags } = require('@oclif/command');
+const fs = require('fs');
+const path = require('path');
+const utils = require('../../util');
+
+class FilesZipCommand extends BoxCommand {
+	async run() {
+        const { flags, args } = this.parse(FilesZipCommand);
+        
+        let fileName = args.name;
+        if (!fileName.includes('.zip')) {
+            fileName += '.zip';
+        }
+
+		let filePath = path.join(flags.destination || this.settings.boxDownloadsFolderPath, fileName);
+        
+        /* eslint-disable no-sync */
+		if (fs.existsSync(filePath)) {
+		/* eslint-enable no-sync */
+
+			let shouldOverwrite = await this.confirm(`File ${filePath} already exists — overwrite?`);
+
+			if (!shouldOverwrite) {
+				return;
+			}
+        }
+ 
+        let output = fs.createWriteStream(filePath);
+        let downloadStatus = await this.client.files.downloadZip(fileName, flags.item, output);
+		await this.output(downloadStatus);
+	}
+}
+
+FilesZipCommand.description = 'Create a zip of multiple files and folders and download it';
+FilesZipCommand.examples = ['box files:zip sample_file.zip --item=file:12421 --item=folder:48291'];
```

Comment:
Can `--item=` take a comma-delimited string of item references instead of/in addition to this?

Reply:
Ya, can change it to a comma if you think it makes more sense.

Reply:
Sorry @sujaygarlanka, I meant to change it to `--items=file:12421,folder:48291` which seems more concise.
Do you think that would make more sense?  I feel like I've seen args done this way more often (in general).  But preference would be to do what is consistent in this app.